src/prefect/server/database/configurations.py

```diff
@@ -176,6 +179,37 @@ def begin_transaction(
 
 
 class AsyncPostgresConfiguration(BaseDatabaseConfiguration):
+    def __init__(
+        self,
+        connection_url: str,
+        echo: Optional[bool] = None,
+        timeout: Optional[float] = None,
+        connection_timeout: Optional[float] = None,
+        sqlalchemy_pool_size: Optional[int] = None,
+        sqlalchemy_max_overflow: Optional[int] = None,
+        statement_cache_size: Optional[int] = None,
+        prepared_statement_cache_size: Optional[int] = None,
+        application_name: Optional[str] = None,
+    ):
+        super().__init__(
+            connection_url=connection_url,
+            echo=echo,
+            timeout=timeout,
+            connection_timeout=connection_timeout,
+            sqlalchemy_pool_size=sqlalchemy_pool_size,
+            sqlalchemy_max_overflow=sqlalchemy_max_overflow,
+        )
+        self.statement_cache_size = (
+            statement_cache_size or PREFECT_API_DATABASE_STATEMENT_CACHE_SIZE.value()
+        )
+        self.prepared_statement_cache_size = (
+            prepared_statement_cache_size
+            or PREFECT_API_DATABASE_PREPARED_STATEMENT_CACHE_SIZE.value()
+        )
+        self.application_name = (
+            application_name or PREFECT_API_DATABASE_APPLICATION_NAME.value()
+        )
```

Comment:
I'd like to phase out using settings in the upper-snake-case format eventually, so can you update this to use `get_current_settings` and get the settings off of the return value? You can import `get_current_settings` directly from `prefect.settings`.

The upside of that is you'll get better autocomplete for available settings.
```suggestion
        settings = get_current_settings()
        self.statement_cache_size = (
            statement_cache_size or settings.server.database.statement_cache_size
        )
        self.prepared_statement_cache_size = (
            prepared_statement_cache_size
            or settings.server.database.prepared_statement_cache_size
        )
        self.application_name = (
            application_name or settings.server.database.application_name
        )
```